### PR TITLE
Corrected minor typos

### PR DESCRIPTION
**Pull request by sushreebarsa**
Updated a few typos in documentation. Also fixing #1038

### CODE REVIEW

File: guides/keras_tuner/getting_started.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def build_model(hp):
 
 """
 There are many other types of hyperparameters as well. We can define multiple
-hyperparameters in the function. In the following code, we tune the whether to
```
Comment (fchollet) [inline guides/keras_tuner/getting_started.py:72]: This is an actual typo

**Review by fchollet — COMMENTED** [review]
Thanks for the PR -- most of these aren't typo but valid American English spellings. Also "a class" is singular. Please restrict the PR to only the first typo and replicate the fix to the md and ipynb files.

File: guides/keras_tuner/getting_started.py (hunk 1 above)
Comment (sushreebarsa) [reply guides/keras_tuner/getting_started.py:72]: Thank you for your response. I have made the changes as required.

**Review by fchollet — COMMENTED** [review]
Please replicate the same fix in the md and ipynb files.

**Comment by sushreebarsa** [comment]
@fchollet I have made the necessary changes as suggested. Thank you!

**Review by MarkDaoust — APPROVED** [review]
Thanks!

**Pull request merged**